multisig/src/weights.rs

```diff
@@ -0,0 +1,140 @@
+// This file is part of Substrate.
+
+// Copyright (C) 2020-2021 Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Weights for pallet_multisig
+//! THIS FILE WAS AUTO-GENERATED USING THE SUBSTRATE BENCHMARK CLI VERSION 2.0.0
+//! DATE: 2020-10-27, STEPS: `[50, ]`, REPEAT: 20, LOW RANGE: [], HIGH RANGE: []
+//! EXECUTION: Some(Wasm), WASM-EXECUTION: Compiled, CHAIN: Some("dev"), DB CACHE: 128
+
+// Executed Command:
+// target/release/framenode
+// benchmark
+// --chain=dev
+// --steps=50
+// --repeat=20
+// --pallet=pallet_multisig
+// --extrinsic=*
+// --execution=wasm
+// --wasm-execution=compiled
+// --heap-pages=4096
+// --output=./multisig/src/weights.rs
+// --template=./.maintain/frame-weight-template.hbs
+
+
+#![allow(unused_parens)]
+#![allow(unused_imports)]
+
+use frame_support::{traits::Get, weights::{Weight, constants::RocksDbWeight}};
+use sp_std::marker::PhantomData;
+
+/// Weight functions needed for pallet_multisig.
+pub trait WeightInfo {
+    fn as_multi_create(s: u32, z: u32, ) -> Weight;
+    fn as_multi_create_store(s: u32, z: u32, ) -> Weight;
+    fn as_multi_approve(s: u32, z: u32, ) -> Weight;
+    fn as_multi_approve_store(s: u32, z: u32, ) -> Weight;
+    fn as_multi_complete(s: u32, z: u32, ) -> Weight;
+}
+
+/// Weights for pallet_multisig using the Substrate node and recommended hardware.
+pub struct SubstrateWeight<T>(PhantomData<T>);
```

Comment:
We probably need to rebenchmark those, or at least scale comparing to other weights.